developer-tools/scaleway-cli/reference-content/keyboard-shortcuts.mdx

```diff
@@ -0,0 +1,34 @@
+---
+meta:
+  title: Scaleway Console CLI keyboard shortcuts
+  description: This page contains the list of keyboard shortcuts to use the Scaleway Console CLI
+content:
+  h1: Scaleway Console CLI keyboard shortcuts
+  paragraph: This page contains the list of keyboard shortcuts to use the Scaleway Console CLI
+tags: devtools cli shortcuts console
+dates:
+  validation: 2023-09-12
+  posted: 2023-05-15
+categories:
+  - devtools
+---
+
+Below is a list of keyboard shortcuts to help you easily use and navigate the Scaleway Console CLI:
```

Comment:
Will it work the same way on all the different operating systems and browser? Ctrl+A is already may in a text field in Windows and Linux. will it work?